userguide/tutorials/deploy_to_kubernetes.adoc

```diff
@@ -0,0 +1,363 @@
+      name: killbill
+    spec:
+      ports:
+        - name: "8080"
+          port: 8080
+          targetPort: 8080
+      selector:
+        service: killbill
+      type: LoadBalancer
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        service: db
+      name: db
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          service: db
+      template:
+        metadata:
+          labels:
+            service: db
+        spec:
+          containers:
+            - env:
+                - name: MYSQL_ROOT_PASSWORD
+                  value: killbill
+              image: killbill/mariadb:0.22
+              name: db
+              ports:
+                - containerPort: 3306
+          restartPolicy: Always
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        service: kaui
+      name: kaui
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          service: kaui
+      template:
+        metadata:
+          creationTimestamp: null
+          labels:
+            service: kaui
+        spec:
+          containers:
+            - env:
+                - name: KAUI_CONFIG_DAO_PASSWORD
+                  value: killbill
+                - name: KAUI_CONFIG_DAO_URL
+                  value: jdbc:mysql://db:3306/kaui
+                - name: KAUI_CONFIG_DAO_USER
+                  value: root
+                - name: KAUI_KILLBILL_URL
+                  value: http://killbill:8080
+              image: killbill/kaui:2.0.9
+              name: kaui
+              ports:
+                - containerPort: 8080
+          restartPolicy: Always
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      labels:
+        service: killbill
+      name: killbill
+    spec:
+      replicas: 1
+      selector:
+        matchLabels:
+          service: killbill
+      template:
+        metadata:
+          labels:
+            service: killbill
+        spec:
+          containers:
+            - env:
+                - name: KILLBILL_DAO_PASSWORD
+                  value: killbill
+                - name: KILLBILL_DAO_URL
+                  value: jdbc:mysql://db:3306/killbill
+                - name: KILLBILL_DAO_USER
+                  value: root
+              image: killbill/killbill:0.22.26
+              name: killbill
+              ports:
+                - containerPort: 8080
+          restartPolicy: Always
+kind: List
+----
+
+In this guide, for running commands against a Kubernetes cluster we would use a command-line tool - *kubectl*.
+
+For more information about the *kubectl* tool, check the following links:
+
+* https://kubernetes.io/docs/reference/kubectl/overview/[Overview of kubectl]
+
+* https://kubernetes.io/docs/tasks/tools/#kubectl[Install kubectl]
+
+*This guide has the following sections:*
+
+* <<deploy_to_local_kubernetes>>
+* <<deploy_to_kubernetes_on_digitalocean>>
+* <<debug_running_pods>>
```

Comment:
If this was a longer document, I'd put this in a Table of Contents. As it is, I would move it to be after the first sentence of the document (This guide explains how to...).